uint16 payloadMask
st_TRSIGNAL[] trSignal
st_BSM[] bsmMsg
st_RSA[] rsaMsg
st_EVA[] evaMsg
st_CAVBSM[] cavbsm
st_RCP[] rcp
